### ISSUE TITLE
keys method return type

### ISSUE THREAD

**Issue by moefar** [ukko/phpredis-phpdoc#50]
Hey guys

I have a situation when I want to get keys with a pattern using `keys()` method. I believe this method sometimes returns another type rather array.

**Comment by ukko** [comment]
Hello, may be method return `bool`? For example `false`, if connection is not stable?

**Comment by scaytrase** [comment]
Yeah, we are also facing this problem. We are getting `boolean` values 

```
    public function getNames(string $pattern = '*'): array
    {
        return $this->redis->keys($pattern);
    }
```

fails with type violation - boolean given instead of array
